xlive/H2MOD/Modules/Console/ConsoleCommands.cpp

```diff
@@ -459,6 +459,7 @@ void ConsoleCommands::handle_command(std::string command) {
 				if (isNum(cstr)) {
 					int peerIndex = atoi(cstr);
 					if (peerIndex == 0) {
+						delete[] cstr;
```

Comment:
can you do a try/finally here? with the delete[] cstr in the finally block

Reply:
ok